backend/notification/notif_categories.js

```diff
@@ -0,0 +1,6 @@
+export const notif_categories = {
```

Comment:
These are already created in https://github.com/izuchukwumba/peered/blob/ba99fb3187dd4f3816abcf7b2aa0c716adbd2961/backend/notification/notif_categories_backend.js#L1, why are you creating duplicates?

Reply:
I updated the file name in an earlier commit in notification-backend which is now merged to main, so it did not update yet. After fixing the merge conflict and deleting the file, it is fixed now.